docker-compose.yml

```diff
@@ -27,3 +27,15 @@ services:
           - "27017:27017"
       volumes:
           - /var/lib/mongodb
+
+    haproxy:
+      image: haproxy:1.7
+      ports:
+          - "80:80"
+      volumes:
+          - /c/projects/proxies/localhost/etc/haproxy:/usr/local/etc/haproxy/
```

Comment:
Does everyone store their projects under this path? Otherwise this won't work...